satip/jpeg_xl_float_with_nans.py

```diff
@@ -0,0 +1,182 @@
+"""Thin wrapper around imagecodecs.JpegXl.
```

Comment:
We could consider moving this file into a separate repository? Maybe it's fine in `satip` for now?

Reply:
I think its fine here, if this is the only place we are using JPEGXL for now